examples/forward/point_mass.py

```diff
@@ -0,0 +1,53 @@
+"""
+Point Mass Forward Modelling on Cartesian Coordinates
```

Comment:
Since the section is "Forward modeling", we can probably remove it from the recipe titles. That will help make them shorter.

Reply:
Ok! You're right!